utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java

```diff
@@ -188,13 +188,21 @@ public void verifyIntegrationWithTask(String task) {
     }
 
     @And("^.*checks? that query \"([^\"]*)\" has \"(\\w+)\" output$")
-    public void checkNumberValuesExistInTable(String query, Integer number) {
-        assertThat(dbUtils.getCountOfInvokedQuery(query)).isEqualTo(number);
+    public void checkNumberValuesExistInTable(String query, int number) {
+        try {
+            OpenShiftWaitUtils.waitFor(() -> dbUtils.getCountOfInvokedQuery(query) == number, 5 * 1000L, 60 * 1000L);
```

Comment:
Maybe it's time to extract this to a reusable method? Seems like we'll be using that pattern more and more